### PR TITLE
Coat of arms updates

### PR DESCRIPTION
**Pull request by murrlipp**
### Description of the change

Update all instances of old Coat of Arms to new version.

### Release notes

Coat of arms updated in:
- Site header
- header component
- Button menu examples
- Date picker examples
- Add to a list pattern

### CODE REVIEW

File: docs/_includes/layouts/base.njk [hunk 1]
```diff
@@ -8,20 +8,20 @@
...
-              href="{{ '/assets/images/moj-apple-touch-icon-167x167.png' | url }}"
+              href="{{ '/assets/images/moj-apple-touch-icon-167x167-2024.png' | url }}"
               sizes="167x167">
         <link rel="apple-touch-icon"
-              href="{{ '/assets/images/moj-apple-touch-icon-152x152.png' | url }}"
+              href="{{ '/assets/images/moj-apple-touch-icon-152x152-2024.png' | url }}"
               sizes="152x152">
         <link rel="apple-touch-icon"
-              href="{{ '/assets/images/moj-apple-touch-icon.png' | url }}">
+              href="{{ '/assets/images/moj-apple-touch-icon-2024' | url }}">
```
Comment (chrispymm) [inline docs/_includes/layouts/base.njk:20]: Think this needs `.png` on the end

File: src/moj/components/header/_header.scss [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 }
 
-.moj-header__logotype-crest {
+.moj-header__logotype-crest-2024 {
```
Comment (chrispymm) [inline src/moj/components/header/_header.scss:34]: we don't need to change the css class name

File: src/moj/components/header/template.njk [hunk 3]
```diff
@@ -14,10 +14,37 @@
...
+        <svg aria-hidden="true" focusable="false" class="moj-header__logotype-crest-2024" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
+
+        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (chrispymm) [inline src/moj/components/header/template.njk:19]: we have an `svg` within an `svg` here - you can remove line 19

File: src/moj/components/header/template.njk [hunk 4]
```diff
@@ -14,10 +14,37 @@
...
+        <svg aria-hidden="true" focusable="false" class="moj-header__logotype-crest-2024" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
```
Comment (chrispymm) [inline src/moj/components/header/template.njk:17]: again, the classname can stay as it was

File: docs/_includes/layouts/base.njk (hunk 1 above)
Comment (robertjmccarthy) [inline docs/_includes/layouts/base.njk:20]: Missing .png?

**Review by robertjmccarthy — COMMENTED** [review]
1 x comment added about suspected missing .png

File: docs/_includes/layouts/base.njk (hunk 1 above)
Comment (murrlipp) [reply docs/_includes/layouts/base.njk:20]: Noted, thanks.

**Review by chrispymm — CHANGES_REQUESTED** [review]
Made some comments on the changes.  

The other thing to note is that the commit message needs to conform to the spec otherwise it won't merge. It should probably be: `fix(crest): Update coat of arms` - this will ensure that the release tool can appropriately increment the version number.

There is a toll on the repo call commitizen that can ensure all your commits follow the format - but you may not have it configured - I can help you sort that out at some point

**Comment by murrlipp** [comment]
> Made some comments on the changes.
> 
> The other thing to note is that the commit message needs to conform to the spec otherwise it won't merge. It should probably be: `fix(crest): Update coat of arms` - this will ensure that the release tool can appropriately increment the version number.
> 
> There is a toll on the repo call commitizen that can ensure all your commits follow the format - but you may not have it configured - I can help you sort that out at some point

Is there a link to the spec anywhere for future reference Chris?

File: src/moj/components/header/template.njk (hunk 4 above)
Comment (murrlipp) [reply src/moj/components/header/template.njk:17]: Noted and amended.

File: src/moj/components/header/_header.scss (hunk 2 above)
Comment (murrlipp) [reply src/moj/components/header/_header.scss:34]: Noted and amended.

File: src/moj/components/header/template.njk (hunk 3 above)
Comment (murrlipp) [reply src/moj/components/header/template.njk:19]: Amended.

**Pull request closed without merging**